packages/augment-api/package.json

```diff
@@ -23,7 +23,8 @@
   "scripts": {
     "clean": "rimraf ./lib",
     "build": "yarn clean && yarn build:ts",
-    "build:types": "yarn generate:defs && yarn generate:meta",
+    "build:types": "yarn generate:defs && yarn generate:meta && yarn build:fixes",
+    "build:fixes": "sed -i.backup -E 's/\\\\bEd25519\\\\b/ed25519/g; s/\\\\bSr25519\\\\b/sr25519/g; s/\\\\bX25519\\\\b/x25519/g; s/\\\\bEcdsa\\\\b/ecdsa/g' src/interfaces/augment-api-tx.ts && rm src/interfaces/augment-api-tx.ts.backup",
```

Comment:
the file we are changing is auto-generated in the previous step, do we even need a backup?

Reply:
It’s unavoidable with MacOS default `sed`. I actually need someone on Mac to confirm this line works :)

Reply:
seems like `\b` does not work on macOS, not even with all the backslashes :( seems like we can only use actual whitespace. `sed -I ''` does not work on linux? (switches off backups on Mac)

Reply:
Wow, MacOS, just wow. Promises extended regex support, but doesn’t support word boundary \b. Well, the sed script was ugly anyway, I’ve replaced it with a bit of javascript in https://github.com/KILTprotocol/sdk-js/pull/588/commits/99fc11618c6ba81bab8437963d2cc939d63cf639

Reply:
Yeah... honestly I'm not even trying to understand the differences between all the different regex dialects bc it's all so inconsistent. Using JS is definitely the way to go to make it work cross-platform (it's also why we use `rimraf` for file removal).
And this means we could even do:
```js
const fixed = source.replace(/\b(Ed25519|Sr25519|X25519|Ecdsa)\b/g, (match) =>
  match.toLowerCase()
)
```

Reply:
Yes, this is even better!